### ISSUE TITLE
Change the logo on start screen

### ISSUE THREAD

**Issue by andreimpircalaboiu** [andreimpircalaboiu/around#7]
We could change the logo on the start screen with the one attached
![banner](https://cloud.githubusercontent.com/assets/11775114/19728332/ab6b4a2a-9b9b-11e6-9661-3cc8babd9610.png)

**Comment by BogdanH95** [comment]
Yeah, it's a good idea since we changed the planet look. It looks greatto me !
